dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -94,11 +113,11 @@ export const TopicFilterBank = ({
 					/>
 				)}
 
-				{availableTopics.slice(0, 5).map((topic) => {
+				{topFiveTopics.map((topic) => {
 					const buttonParams = `${topic.type}:${topic.value}`;
-					const isActive = selectedTopics
-						? selectedTopics.includes(buttonParams)
-						: false;
+					const isActive =
+						selectedTopic !== undefined &&
```

Comment:
The type of the `selectedTopic` is `Topic | undefined` so I don't think it can be anything other than these 2 types, right?

further up in the code I'm doing this: `const selectedTopic = selectedTopics?.[0];` which will return `Topic | undefined`

Reply:
Ah ok sure, I'd missed that bit

Reply:
I just changed it to `!!selectedTopic && isEqual(selectedTopic)(topic);` . This is hopefully nicer to read?